sycl/include/sycl/handler.hpp

```diff
@@ -3570,6 +3581,12 @@ class __SYCL_EXPORT handler {
             "A local accessor must not be used in a SYCL kernel function "
             "that is invoked via single_task or via the simple form of "
             "parallel_for that takes a range parameter.");
+      if (Kind == detail::kernel_param_kind_t::kind_work_group_memory)
+        throw sycl::exception(
+            make_error_code(errc::kernel_argument),
+            "A local accessor must not be used in a SYCL kernel function "
```

Comment:
Probably it's better to use "workgroup memory" instead of "local accessor" in this error message.

Reply:
Oops, indeed. I blindly copied the message from the local accessor case.
Thanks!